en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md

```diff
@@ -0,0 +1,214 @@
+
+| parameter name | parameter description                         |
+|----------------|-----------------------------------------------|
+| code           | short textual error code indicating the error |
+| message        | description of the error                      |
+
+
+You have successfully implemented Choreo's Managed Authentication for your web application. Your next step involves creating a web application component in Choreo, enabling Managed Authentication for the component, and subsequently deploying it.
+
+## Step 2: Enable Managed Authentication
+
+To ensure that your web application functions seamlessly with Managed Authentication, it is essential to enable Managed Authentication for your web application component within Choreo.
+
+There are two ways that you can enable Managed Authentication for your Web Application component Choreo: 
+- At component creation
+- At component deployment
+
+### Enable Managed Authentication at component creation
+
+When creating a web application component, click the toggle **Managed authentication with Choreo** under **Authentication**  to enable Managed Authentication.
+
+!!! note
+    Managed Authentication is currently available only for React, Angular, and Vue.js buildpacks.
+
+### Enable Managed Authentication at component deployment
+
+1. Go to the **Deploy** view of your component.
+2. Click **Authentication Settings** on the **Set Up** card.
+3. Enable/Disable Managed Authentication using the toggle in the sidebar.
+
+## Step 3: Configure Managed Authentication
+
+To configure the necessary paths and scopes for managed authentication, follow the steps below:
+
+1. Go to the **Deploy** view of your component.
+2. Click **Authentication Settings** on the **Set Up** card.
+2. Configure the following fields:
+
+
+| Option            |  Description      | Default           |
+| ----------------- | ----------------- | ----------------- |
+| Post Login Path   | This represents the relative path that the application will be redirected to following a successful login. In your code, you will need to implement the necessary logic to handle the userinfo cookie set by Managed Authentication. See [ Handling user info post-login](#handling-user-info-post-login)       | /                      |
+| Post Logout Path  | The relative path to which Choreo redirects after a successful logout.  | /                      |
+| Error Path        | The relative path to which Choreo redirects to if an error occurs during a redirection-based flow (i.e. Login, Logout). See [Custom Error Page](#custom-error-page)             | Builtin error page     |
+| Additional Scopes | All additional scopes required by the web application. The following scopes will be added by default. `openid`, `profile`, `email`, and scopes required to invoke subscribed APIs.               | none                   |
+
+
+## Step 4: Manage OAuth Keys
```

Comment:
Can we say we auto-generate keys for web application when in-built IDP is used

Reply:
Resolved with [c5726a1](https://github.com/wso2/docs-choreo-dev/pull/1183/commits/c5726a15bc558bf96b3fb05ab1f27c4838baf301)